app/types/types.ts

```diff
@@ -25,6 +25,11 @@ export interface ColDataCardDef<T extends Record<string, unknown>> {
    * If the column is editable
    */
   editable?: boolean;
+  /**
+   * If the field should only be available in the exported table data -
+   * only affects when column is used in a table config
+   */
+  exportOnly?: boolean;
```

Comment:
new prop that enables columns to be hidden from the Table component

Reply:
✨